.github/workflows/new-release-tag-docker.yml

```diff
@@ -35,6 +35,17 @@ jobs:
         run: |
           git tag -a ${{ github.event.inputs.release_number }} -m "[bot] release ${{ github.event.inputs.release_number }}"
           git push origin ${{ github.event.inputs.release_number }}
+      - name: Install Helm
+        run: |
+          curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+          chmod 700 get_helm.sh
+          ./get_helm.sh
+      - name: Package Helm chart
```

Comment:
Before this step, you'd need to add the repos, such as:

```
- name: Configure Helm repos
        run: |-
             helm repo add stable https://charts.helm.sh/stable
             helm repo add bitnami https://charts.bitnami.com/bitnami
             helm dependency list ./helm/defectdojo
             helm dependency update ./helm/defectdojo
```

or the build will fail.